### ISSUE TITLE
Syncthing service never starts

### ISSUE THREAD

**Issue by 1024mb** [Catfriend1/syncthing-android#664]
The android app starts fine but the service doesn't. I get this on the logs after restarting the app:

```
17:48:05I/sh       type=1400 audit(0.0:35176): avc: denied { read } for name="/" dev="sda13" ino=2 scontext=u:r:untrusted_app:s0:c126,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=1
17:48:05I/sh       type=1400 audit(0.0:35177): avc: denied { open } for path="/" dev="sda13" ino=2 scontext=u:r:untrusted_app:s0:c126,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=1
```

I'm also getting this like thousands of the same text:

```
17:48:04W/PollWebGuiAvailableTask Unexpected error while polling web gui
17:48:04W/PollWebGuiAvailableTask Unexpected error while polling web gui
17:48:04W/PollWebGuiAvailableTask Unexpected error while polling web gui
```

 The app then shows a notifocation suggesting that I may have another app using the same port but I do not.

**Comment by Catfriend1** [comment]
Hi, do you have both apps official + fork installed?

**Comment by 1024mb** [comment]
> Hi, do you have both apps official + fork installed?
 
Hey, no, I didn't have the official app installed, I only had the fork installed from F-droid repo.

I've tried right now to open the fork again and this time it did started the service.

Now I do have the official app installed on my phone but I haven't open it, I installed it just in case. In fact, I installed the official app some minutes after opening this issue and, at that time, the fork continued to show me that error with that app installed, so I don't know what it was causing the problem.
